data-reconciliation-app/docs/data-reconciliation.md

```diff
@@ -0,0 +1,69 @@
+# What is data reconciliation means in the context of the project?
+
+## _Reconciliation in finance_
+
+Reconciliation in finance is the process of comparing different business entities, accounts, amounts, data points, etc. to find out if they are matching or to explain why there are differences between them.
+
+Data Reconciliation is important because it will address a range of issues such as:
+
+- Missing records
+- Missing values
+- Incorrect values
+- Duplicated records
+- Badly formatted values
+- Broken relationships across tables or systems
+
+![reconciliation diagram](data-reconciliation.png)
+
+## Use case
```

Comment:
I would recommend we remove this section until we identify the specific use case :)

Reply:
done